src/reducers/fileTree.tsx

```diff
@@ -199,14 +199,18 @@ export const getRelativePath = ({
 
 type GoToRelativeFileParams = {
   _getRelativePath?: typeof getRelativePath;
+  _viewVersionFile?: typeof viewVersionFile;
   currentPath: string;
   pathList: string[];
   position: RelativePathPosition;
   versionId: number;
 };
 
 export const goToRelativeFile = ({
+  /* istanbul ignore next */
```

Comment:
Why do we need these comments? We are exercising these via the tests. Is there some issue with the way the `thunkTester` works that makes it so that they seem not covered even though they are?

Reply:
We never test the real implementation hence some missing code coverage. This comment should tell the coverage tool to ignore the default assignation.

Reply:
Maybe we should have a test that uses the real implementation? Although I don't know what that test would do. 🤷‍♂️